content/integrations/lambdatest-integration.md

```diff
@@ -0,0 +1,54 @@
+---
+title: LambdaTest integration
+description: How to integrate your workflows with LambdaTest using codemagic.yaml
+weight: 6
+---
+
+**LambdaTest** is a cloud-based cross browser testing tool that allows you to perform cross browser testing for mobile applications. You can perform both manual and automated cross browser testing using LambdaTest. More information can be found here. 
```

Comment:
You have mentioned "cross browser" three times in this sentence. Surely we are testing mobile applications, not web apps on multiple browsers??

Reply:
Done!